### ISSUE TITLE
Transparent webpage.bg causes completion to be shown behind website

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#5948]
**Version info**:
qutebrowser v1.14.1
Git commit: bf65a1db0 on master (2020-12-07 20:14:23 +0100)
Backend: QtWebEngine (Chromium 83.0.4103.122)
Qt: 5.15.2

With picom vgit-dac85

**How to reproduce**
```
qutebrowser --temp-basedir -s mainwindow.transparent true -s colors.webpage.bg "#22ff0000" about:blank ":set-cmd-text :"
```

![image](https://user-images.githubusercontent.com/625793/101514661-ce505b00-397d-11eb-80a3-9b920e705fe5.png)

**Comment by dogeystamp** [comment]
**Version info**:

qutebrowser v2.5.4
Git commit: 7724f7295 on main (2023-07-24 23:04:51 +1200)
Backend: QtWebEngine 6.5.2, based on Chromium 108.0.5359.220 (from api)
Qt: 6.5.2 (compiled 6.5.1)

With picom vgit-b700a

**How to reproduce**
```
qutebrowser --temp-basedir -s window.transparent true -s colors.webpage.bg "#22ff0000" about:blank ":set-cmd-text :"
```

![image](https://github.com/qutebrowser/qutebrowser/assets/61116261/cc00d668-a203-409e-b84f-6b8c30b1001d)

The bug no longer happens on this version of Qutebrowser.

**Comment by dogeystamp** [comment]
I spoke too soon.
![image](https://github.com/qutebrowser/qutebrowser/assets/61116261/1c63d0a5-7c2e-4872-9e56-df75a9290d69)
This still causes a bug:
```
qutebrowser --temp-basedir -s window.transparent true -s colors.webpage.bg "#22ff0000" https://nocss.club ":set-cmd-text :"
```
Oddly enough, if I then go to https://github.com, then go back to nocss club, this happens:
![image](https://github.com/qutebrowser/qutebrowser/assets/61116261/73186e90-8758-46c5-9652-2420eaf0c20c)

**Comment by The-Compiler** [comment]
Also happens with qb v3.1.0 and Qt 6.6.1 according to #8025

**Comment by johnazule** [comment]
Has there been any progress with this? I am new to the project and unsure where to jump in with this

**Comment by The-Compiler** [comment]
No, and same here - it might well be a Qt/QtWebEngine rather than a qutebrowser issue.

**Comment by ElYaiko** [comment]
Still happening, hints too.
